src/BuildAnalyzer/Analyzers/FileCompilations/FileCompilationsAnalyzer.cpp

```diff
@@ -48,6 +48,13 @@ const wchar_t* FileCompilationsAnalyzer::GetFilePath(const CppBI::Activities::Co
         return compilerPass.InputSourcePath();
     }
 
-    assert(compilerPass.OutputObjectPath() != nullptr);
-    return compilerPass.OutputObjectPath();
+    // ms turns out VS2019 OutputObjectPath can be null too
+    // fallback on something
+    if (compilerPass.OutputObjectPath() != nullptr)
+    {
+        return compilerPass.OutputObjectPath();
+    }
+
+    return L"Unknown";
+
```

Comment:
These lines look like they're part of https://github.com/MetanoKid/cpp-build-analyzer/pull/51.
Can you please move them away so both Pull Requests aren't mixed?

Reply:
Removed.
Sorry about this.